### ISSUE TITLE
[UI] container window style is not standard (+ lacks Aero support)

### ISSUE THREAD

**Issue by Drugoy** [cshnik/TaskSpace#4]
There's something wrong with the styles applied to the new empty container windows of TaskSpace:
<img src="https://i.imgur.com/7Jrx89g.png">
(disregard the extra button in the title bar created by TeamViewer).

I'm on w7sp1x64 with Aero.
1. That window doesn't have Aero effects.
2. Window control buttons in the title bar are not from Aero theme.
3. Not all the corners are rounded.
4. Top corners are rounded but have black background underneath them.
5. Left clicking the icon in the top left corner of the window ("favicon") seems to re-apply some styles or redraw something, since the teamviewer's extra title bar button blinks once or twice after the click).

**Comment by Drugoy** [comment]
I tried to compare WinExplorer's window's style and exstyle to taskspace container window's style and exstyle:

```
WinExplorer:
Style: 0x14CF0000
ExStyle: 0x00000100

TaskSpace:
Style: 0x16CF0000
ExStyle: 0x00040308
```

Is that intentional? Why?

**Comment by Drugoy** [comment]
Also, due to that styling - the window gets redrawn noticeably slow (just try to resize TaskSpace container's window with the mouse by dragging by the window's corner and you'll see black lines slowly disappearing outside the window, when you shrink it's size).

**Comment by cshnik** [comment]
That's how custom drawing for TaskSpace frame is implemented. Aero style was never assumed to be supported. There are multiple things to improve in this area. Probably, such non-functional UI issues should be deffered.

**Comment by Drugoy** [comment]
Well, the thing is not that TaskSpace containers lack aero itself, but that (probably) due to non-standard styles the window gets redrawn slowly and they blink too much.

**Comment by cshnik** [comment]
Blinking of the frame(black rectangles at the right and bottom sides) is the result of bug somewhere in the custom drawing code. This can be published as a separate issue for easy tracking.

**Comment by Drugoy** [comment]
Okay, let's split the issue into two: I'll keep this one for non-standard theming being used (and thus no aero support too) and just opened issue #8 for slow redrawing/blinking.

**Comment by cshnik** [comment]
Fixed in 0.1.6.0.
